resources/quality/anycubic_kobra_go/anycubic_kobra_go_abs.inst.cfg

```diff
@@ -0,0 +1,33 @@
+[general]
+version = 4
+name = Suggested parameters for ABS_V1.0
```

Comment:
I think you are misunderstanding the use of these qualities. The names should be things like Fast/Normal/Fine and you should use material = generic\_pla/generic\_abs in the metadata.

This way when the user selects generic\_pla/generic\_abs as their material this quality will show for them.

Reply:
FYI these names are copied from the profiles that get provide with the printer, will change it when I'm home again

Reply:
Also the other Kobra qualities seem similar to the one I commited, are they wrong/too old?